# inputs
float64 stopping_distance
---
# outputs
bool set
